pkg/config/admission/config.go

```diff
@@ -0,0 +1,43 @@
+package admission
+
+import (
+	"strings"
+
+	"github.com/horizoncd/horizon/pkg/admission/models"
+)
+
+type FailurePolicy string
+
+func (f FailurePolicy) Eq(other FailurePolicy) bool {
+	return strings.EqualFold(string(f), string(other))
+}
+
+const (
+	FailurePolicyIgnore FailurePolicy = "ignore"
+	FailurePolicyFail   FailurePolicy = "fail"
+)
+
+type ClientConfig struct {
+	URL      string `yaml:"url"`
+	CABundle string `yaml:"caBundle"`
+	Insecure bool   `yaml:"insecure"`
+}
+
+type Rule struct {
```

Comment:
Can you give some examples about these configurations written in the description of the PR?